js/components/icon.js

```diff
@@ -25,14 +25,16 @@ Icon.propTypes = {
   icon: React.PropTypes.string,
   skinConfig: React.PropTypes.object,
   className: React.PropTypes.string,
-  style: React.PropTypes.object
+  style: React.PropTypes.object,
+  dataTip: React.PropTypes.string
```

Comment:
Would this be clearer if we named it `toolTip`? I realize that `data` is needed for the attribute, but the parameter would probably be better understood as `toolTip`.